### DISCUSSION TITLE
Height Range Modifiers Override Print Settings?

### DISCUSSION THREAD

**Discussion by UD551** [supermerill/SuperSlicer#2252] [Q&A]
Hello Super Slicer Team,
Thank you for providing such a great software to use.  I really appreciate the team's hard work. 

In Regards to Super Slicer's newest version, I was wondering if using a height range modifier will override the preset print settings? 
For example, if I am using a height range modifier to modify the solid infill print speed, in a certain height range, to a value higher than the "modifiers" infill print speed. Will it override the preset solid infill speed value in the "modifiers" section of the print settings? 
In other words, I am trying to use a height range modifier to change the solid infill print speed on the first layer. However, I then realized that there is a "modifiers" speed print setting that might be limiting my infill speed value. I was just wondering if using the height modifiers will override the 'modifiers' value? 

Sorry for complicating it. I hope I was able to convey my issue properly. 
Thank you for the help.

**Comment by supermerill** [comment]
yes, the modifier replace the settings in its area.

**Reply by UD551** [reply]
With all due respect, I think you are incorrect. Let me explain:

I tried changing the first layer solid infill speed by manipulating it using **height range modifiers** in that area. However, I noticed that the speed, when printing, is _not changing_ even though I chose a higher value for it. 
After several trials and error, I noticed that the solid infill speed was limited by the specified value in the modifies for the first layer. You can find this value when you follow this path;  Print settings > Speed > Modifiers > First Layer Speed > Infill ; I noticed that this infill speed is limiting my maximum speed, since when I chose higher print values in the height range modifiers, the actual print speed didn't adjust and was limited by that 'infill' value. 

In conclusion, I believe that the parameters used in the height range modifiers are not 'overriding' the preset value in the print settings. 
I hope this explains my perspective on the issue. 

Thank you for your time!!

**Reply by neophyl** [reply]
This sounds like it needs an issue opening rather than a discussion. As when opening any issue could you please attach a zipped up 3mf project file so that the issue has a chance of being  reproduced. If the dev can’t reproduce it then they stand little chance of fixing it.

**Reply by supermerill** [reply]
> I tried changing the first layer solid infill speed by manipulating it using height range modifiers in that area. However, I noticed that the speed, when printing, is not changing even though I chose a higher value for it.

for first layer, be careful to also set the "first layer speed" -> max & infill to 100%, or it will be modified / erased
